CLAUDE.md

```
@@ -0,0 +1,9 @@
+  # Kiwi-106 Web Interface Project
```

markdownlint-cli2 (0.17.2)

Headings must start at the beginning of the line

(MD023, heading-start-left)